picard/util/alphabet_detector.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# The MIT License (MIT)
+#
+# Copyright (C) 2015 Eli Finkelshteyn
+# Copyright (C) 2021 Vladislav Karbovskii
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import unicodedata as ud
+from collections import defaultdict
```

Comment:
You need to re-order imports, `isort` can help (see https://github.com/metabrainz/picard/blob/master/CONTRIBUTING.md#coding-style)

```patch
-import unicodedata as ud
 from collections import defaultdict
+import unicodedata as ud
```